src/gadgets/mpt_update.rs

```diff
@@ -1480,9 +1480,7 @@ fn configure_balance<F: FieldExt>(
             SegmentType::AccountLeaf3 => {
                 cb.assert_equal("direction is 1", config.direction.current(), Query::one());
                 cb.condition(
-                    config
-                        .path_type
-                        .current_matches(&[PathType::Common, PathType::ExtensionOld]),
```

Comment:
Why remove the `ExtensionOld` case?

Reply:
ExtensionOld means that the old path is being extended, i.e. the old trie contains the account, but the new one does not. This corresponds to an account deletion, which is not supported in our zkevm.